### PR TITLE
Fix selection and overflow when zero rows

### PR DESCRIPTION
**Pull request by platypii**
Fixes two bugs that occur when numRows = 0.

1. nbsp turns into ellipses because the row number column gets too narrow
2. a bug in selection that would cause error if you tried to "select all" on an empty dataframe

https://github.com/user-attachments/assets/eabd1f84-ad33-4ada-99ff-8c9100c520e1

A separate question is why does it get so narrow? The css for the row numbers should have `min-width:48px` but I think it gets overridden by the column sizing. @severo

### CODE REVIEW

File: src/components/HighTable/HighTable.tsx [hunk 1]
```diff
@@ -478,7 +478,7 @@ export function HighTableInner({
                   ariaColIndex={1}
                   ariaRowIndex={1}
                   ref={tableCornerRef}
-                >&nbsp;</TableCorner>
+                />
```
Comment (severo) [inline src/components/HighTable/HighTable.tsx:483]: Yes, good idea

**Review by severo — APPROVED** [review]
Good fix for this corner case.

I added a unit test + improved a bit the CSS in https://github.com/hyparam/hightable/pull/217 (against this PR). Feel free to merge if it works for you
